api/create_tables.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE IF NOT EXISTS cs348db;
```

Comment:
add some table deletion sql at the top of the script, in case this script is ran multiple times and changes are made in ebtween runs

Reply:
I put the drop tables in a separate sql file, if you want I can combine them.